apps/omg/lib/omg/wire_format_types.ex

```diff
@@ -59,6 +64,12 @@ defmodule OMG.WireFormatTypes do
   @spec tx_type_modules() :: module_map_t()
   def tx_type_modules, do: @tx_type_modules
 
+  @doc """
+  Returns the tx type that is associated with the given module
+  """
+  @spec module_tx_types() :: module_map_t()
```

Comment:
I know you didn't name this typespec... but it's a bit uncommon. Could we change this to something more meaningful?